### PR TITLE
Munievents

### PR DESCRIPTION
**Pull request by magdasalatka**
Interactive visualization of municipality events

### CODE REVIEW

**Comment by l00mi** [comment]
@magdasalatka lets rather make a new file municipality-event-per-municipality.html and i will make a new entry in the menu.

**Comment by l00mi** [comment]
https://github.com/zazuko/data.admin.ch/blob/master/config.json#L23

**Comment by l00mi** [comment]
and https://github.com/zazuko/data.admin.ch/blob/master/config.json#L76

**Comment by magdasalatka** [comment]
@l00mi like this?

**Review by l00mi — APPROVED** [review]

**Comment by l00mi** [comment]
exactly! thanks

File: config.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "/search/municipalities/": "cwd:views/search/municipalities.html",
     "/search/offices/": "cwd:views/search/offices.html",
     "/visualizations/map/": "cwd:views/visualizations/map.html",
-    "/visualizations/municipality-events/": "cwd:views/visualizations/municipality-events.html"
```
Comment (sandhose) [inline config.json:23]: @magdasalatka I think you forgot to rename that file :slightly_smiling_face:

**Pull request merged**